client/dashboard/site-overview/trend-comparizon-badge.tsx

```diff
@@ -1,6 +1,6 @@
 import { __experimentalHStack as HStack, Icon } from '@wordpress/components';
 import { arrowDown, arrowUp } from '@wordpress/icons';
-import CoreBadge from 'calypso/components/core/badge';
+import CoreBadge from '../badge';
```

Comment:
I don't understand this change. I think `calypso/components/core/badge` should be ok to import. 

In my mind, we should check what we import, if the component is authorized within the "allow list" it should be ok, no need to duplicate it.

Reply:
There are a lot of things in Calypso that are good to use, and I don't think we should forbid using them. The rule is here is just to "remind" developers that they should first check that what they're importing is correct or not. I don't see this as breaking the rules.

Reply:
The alternative that I would be ok with is to actually move the component and update its usage but not duplicate.

Reply:
I've adjusted it